service/src/test/java/bio/terra/cbas/runsets/monitoring/TestSmartRunsPoller.java

```diff
@@ -60,6 +63,12 @@ public class TestSmartRunsPoller {
           .setDateFormat(new StdDateFormat())
           .setDefaultPropertyInclusion(JsonInclude.Include.NON_ABSENT);
 
+  private CromwellService cromwellService = mock(CromwellService.class);
+  private RunDao runsDao = mock(RunDao.class);
+  private WdsService wdsService = mock(WdsService.class);
+  private SmartRunsPoller smartRunsPoller =
+      new SmartRunsPoller(cromwellService, runsDao, wdsService, objectMapper);
+
```

Comment:
The problem with doing this is that now the exact same mock objects are shared between all the tests, meaning that any mock configuration is shared, and any "did x get called" checks might be picking up method calls from previously run tests.

My suggestion would be to make these global variables with default values (ie, set them all to `null`, and then use a `before` hook to create the instances for each of them afresh for each test case.

Reply:
Ah I see. Yeah I can implement your suggestion 👍